server/endpoint/pom.xml issue with org.json:json dependency

## This is a...

Trying to build syndesis:master and getting the following in server/endpoint.

```
[INFO] --- maven-dependency-plugin:3.1.1:analyze-only (basepom.default) @ server-endpoint ---
[WARNING] Used undeclared dependencies found:
[WARNING]    org.json:json:jar:20140107:compile
[INFO] ------------------------------------------------------------------------
```
```
[INFO] Server :: Inspector ................................ SUCCESS [ 13.996 s]
[INFO] Server :: Credential ............................... SUCCESS [ 24.949 s]
[INFO] Server :: Endpoint ................................. FAILURE [ 16.563 s]
```

I'm not sure how to respond to this - if I exclude org.json:json from openshift-client (which is bringing it in), then I get an error 

```
[INFO] --- dependency-management-maven-plugin:0.11:analyze (basepom.default) @ server-endpoint ---
[ERROR] Exclusions must be removed for managed dependency io.fabric8:openshift-client:jar
```

If I add a dependency to server/endpoint for org.json:json, I get a complaint that android-json has classes that are duplicated from org.json:json.




## Description

--- a/app/pom.xml
+++ b/app/pom.xml
@@ -2462,8 +2462,18 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <dependency>
+        <groupId>io.fabric8</groupId>
+        <artifactId>openshift-client</artifactId>
+        <version>${kubernetes.client.version}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>org.json</groupId>
+            <artifactId>json</artifactId>
+          </exclusion>
+        </exclusions>
+      </dependency>
     </dependencies>
-
   </dependencyManagement>
 
   <!-- Metadata need to publish to central -->

